scoresheets/StoringGroceries.tex

```diff
@@ -1,8 +1,11 @@
 
-The maximum time for this test is 1+3 minutes.
+The maximum time for this test is 1+2+2 minutes.
 The first minute is for the robot to open the Cupboard's door on its own.
 If the robot is not able to open the door within that minute, it will be opened by the referee. 
 In case the robot opens the door within the minute, the robot has a small time advantage. 
+Then, the robot has 2 minutes to recognize objects and manipulate items.
+If an item is manipulated within that time, 2 more minutes are added to the test time. 
```

Comment:
I'm fine with the robot looking and driving around (!= inoperative) to get a better view.